src/pyodide/internal/python.ts

```diff
@@ -94,7 +99,7 @@ function instantiateWasm(wasmImports, successCallback) {
  * This is a simplified version of the `prepareFileSystem` function here:
  * https://github.com/pyodide/pyodide/blob/main/src/js/module.ts
  */
-function prepareFileSystem(Module) {
+function prepareFileSystem(Module: Module): void {
```

Comment:
Doesn't have to be done in this PR but I really think we should rename the args to lowercase `module`.